app/controllers/users_controller.rb

```diff
@@ -1,10 +1,17 @@
 class UsersController < ApplicationController
-  def show
-    @user = User.find_by id: params[:id]
+  before_action :find_user, except: [:new, :create, :index]
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
  before_action :find_user, except: [:new, :create, :index]
```

để gọi method `logged_in_user` chị nghĩ dùng `except` luôn thay vì `only` nhé

ngoài ra, chị suggest dùng `%i()` thay vì `[]` nhé